snippets/assertions.json

```diff
@@ -0,0 +1,72 @@
+{
+	"t.pass()": {
+		"prefix": "t.pass",
+		"body": "t.pass(${1:'${2:message}'});",
+		"description": "Passing assertion."
```

Comment:
Is it normal to end in a `.` for snippets? Otherwise I think we should remove it.